### ISSUE TITLE
Decide upon a name for the project!

### ISSUE THREAD

**Issue by garrettw** [redditPHP/redditPHP#1]
Ideally we need to check that a URL is available for it too.

If everyone comes up with two suggestions:

- @weirdlooop entry 1
- @weirdlooop entry 2
- @garrettw entry 1
- @garrettw entry 2
- @Kameloh entry 1
- @Kameloh entry 2

Then if we all have 2 votes each where you can't vote on your own - the winner will be the name!

Once this is done I will update the project files and repo accordingly and purchase the domain

**Comment by garrettw** [comment]
### Capybara ⚠ / CapybaraPHP ✅
From a marketing image standpoint, we could go crazy/silly/meme-y/trendy with this one since that's basically where it came from.
Color scheme: bold, strong, bright colors. But also the brown of a capybara.
Domain availability: see emoji above

### Alternative
Pick a different animal.

**Comment by garrettw** [comment]
### Gusty / GustyPHP
Meaning: In Spanish, "me gusta" = "I like". In Esperanto, "ĝusta" = "correct".
Slogan: Feel the pleasant breeze -- or -- Feel the winds of change
Light color scheme: calming light blue/off-whites
Dark color scheme: dark blue/gray, stormy
Domain availability: ✅

### Alternatively -- Satisphy ✅ / Satis 🆗 / SatisPHP ✅
Meaning: it will satisfy you. Also, in Esperanto, "I like" = "mi ŝatas". (sorry lol, I'm learning Esperanto currently)
Light color scheme: should be happy, bright, maybe some yellow like the sun, or green like plants growing in spring
Dark color scheme: should be luxurious, smooth, elegant, sleek, comfortable, modern, sexy, fancy but not over the top
Domain availability: see emoji above

**Comment by weirdlooop** [comment]
**Thetic / TheticPHP / TheticMVC**

Meaning: 

- Presented dogmatically; arbitrarily prescribed. 
- (Poetry) (in classical prosody) of, bearing, or relating to a metrical stress
- positive and arbitrary; prescriptive

Domain availability: ✅

**Comment by garrettw** [comment]
> **Thetic / TheticPHP / TheticMVC**

This makes me think of Scientology with its “thetans” 😆
Also I am against having MVC/etc. in the name.

**Comment by Kameloh** [comment]
### Cosmic ❌ / CosmicPHP 🆗 (.com taken but expired)

'cause space is pretty cool!

### Comet ❌ / CometPHP ✅

Again, more space stuff!

**Comment by weirdlooop** [comment]
# Raphia / RaphiaPHP

So bit of a random one, but the word "hypergraphia" meaning: 

_Hypergraphia is a behavioral condition characterized by the intense desire to write or draw. Forms of hypergraphia can vary in writing style and content._

I used that word in an *unscrambler* to make up other words and Raphia was one of them.

Domain availability: ✅ (all but raphia.com)


## Alternatives

Check out other words that can be made [https://wordunscrambler.me/unscramble/hypergraphia](https://wordunscrambler.me/unscramble/hypergraphia)

**Comment by garrettw** [comment]
> **Thetic / TheticPHP / TheticMVC**
> 
> Meaning:
> 
>     * Presented dogmatically; arbitrarily prescribed.
> 
>     * (Poetry) (in classical prosody) of, bearing, or relating to a metrical stress
> 
>     * positive and arbitrary; prescriptive

I thought a bit more about this one and decided to toss in a 3rd idea: **Skriba**
I got this by translating "prescriptive" to Esperanto, which is "preskriba", and chopped off the "pre". It is true that "skriba" actually translates as "written" -- but I think it just looks better that way.
Domain availability: ✅

**Comment by garrettw** [comment]
> CosmicPHP
> 
> 'cause space is pretty cool!
> 
> CometPHP
> 
> Again, more space stuff!

A 4th idea along these lines: **Orbit ❌ / OrbitPHP ✅**

**Comment by garrettw** [comment]
I don't think we really have to limit ourselves to 2 ideas per person. Also I propose we run this poll using ranked-choice voting, and I can manage it. So basically once all options are in, we will each comment our favorites in order of preference.

In any case, ~~I'll check domain availability on everything now~~ I've checked domain availability on everything and added that info to all the comments.

**Comment by garrettw** [comment]
My choices as it stands now:

1. Raphia
2. Skriba
3. Satisphy
4. SatisPHP
5. Satis
6. CapybaraPHP
7. OrbitPHP
8. Gusty
9. CosmicPHP
10. CometPHP
11. Thetic

**Comment by Kameloh** [comment]
I think those are some good choices so far -- do you think it's enough to start voting?

**Comment by weirdlooop** [comment]
Ye good enough to start voting I would say

**Comment by weirdlooop** [comment]
Where is the vote?

**Comment by garrettw** [comment]
@weirdlooop mine is right up there ^. Where is anyone else's?

**Comment by garrettw** [comment]
@weirdlooop @Kameloh at this point I'm honestly wondering if you guys still have any interest in this, as I haven't heard anything from either of you when I've reached out, and haven't seen any activity here in the repo either.
